osmaxx-py/osmaxx/excerptexport/__init__.py

```diff
@@ -1,5 +1,3 @@
-from osmaxx.excerptexport.forms.excerpt_order_form_helpers import get_country_choices
-
 __all__ = [
```

Comment:
`__all__` and the assignement should be removed as well, since this module doesn't expose this anymore.